.github/workflows/framework-tests.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
           PEBBLE: /tmp/pebble
 
       - name: Run Real pebble tests
-        run: tox -e unit -- -k RealPebble
+        run: tox -e unit -- test/test_real_pebble
```

Comment:
`.py`?

Reply:
Ha, yes, sorry. I realised I did that and searched for it to fix it up where I had it wrong, and forgot I'd need a `--hidden` for `rg` :joy: